src/vs/vscode.d.ts

```diff
@@ -6079,6 +6079,30 @@ declare module 'vscode' {
 
 	}
 
+	/**
+	 * The event that is fired when there is a change in [tree view's selection](#TreeView.selection)
+	 */
+	export interface TreeViewSelectionChangeEvent<T> {
+
+		/**
+		 * Selected elements.
+		 */
+		selection: T[];
```

Comment:
add `readonly` ~~and `ReadonlyArray<T>`~~?

Reply:
This was there in last milestone. Hope the change will not break.

Reply:
just `readonly` should be enough and non breaking